pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/blocks/TransferableBlock.java

```diff
@@ -156,7 +157,7 @@ public boolean isErrorBlock() {
 
   public byte[] toBytes()
       throws IOException {
-    return _dataBlock.toBytes();
+    return getDataBlock().toBytes();
   }
```

Comment:
Add a new API call and rename the current one:
```
public byte[] getDataBlock() {
  // ...
}
public List<byte[]> getDataBlockTrunk() {
  // ...
}
```

Reply:
Done